amplify/backend/auth/cognitoa7d29d9d/cognitoa7d29d9d-cloudformation-template.yml

```diff
@@ -508,6 +508,20 @@ Resources:
     DependsOn: IdentityPool
   
 
+  StripePaymentApplicationAccessPolicy:
```

Comment:
That's gonna break in future releases, specially if someone uses amplify auth update. 

My suggestion for now is either option:

1.  Remove it and implement our own API GW integration with Stripe
2. Improve integration and update the Stripe SAR App we use
2.1. [Ask Alex or make a PR to enforce IAM authorizers via parameters](https://github.com/simalexan/api-lambda-stripe-charge)
2.1. Add Payment type in the GraphQL Schema
2.2. Create a Payment API data source connecting to Alex's SAR App
2.3. Create `verifyPayment` mutation with a custom resolver  to make a POST to Alex via IAM Auth
2.4. Update front-end to use Amplify so we can use Cognito credentials